StripePaymentSheet/StripePaymentSheet/Source/Helpers/PaymentSheetLinkAccount.swift

```diff
@@ -47,6 +47,8 @@ class PaymentSheetLinkAccount: PaymentSheetLinkAccountInfoProtocol {
         // Inline, email-prefilled
         case implied_v0_0 = "implied_consent_withspm_mobile_v0_0"
 
+        // Clicked button in separate Link sheet
+        case clicked_button_mobile = "clicked_button_mobile"
```

Comment:
Seems like we should version this similar to the other ones? e.g.:
clicked_button_mobile_v0